scripts/setup_build/pre-commit

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright (c) Microsoft Corporation
```

Comment:
nit: The same file exists under scripts folder (https://github.com/microsoft/ebpf-for-windows/blob/main/scripts/pre-commit). Is this a duplicate?

Reply:
Oops.